composer.json

```diff
@@ -69,7 +69,8 @@
 		],
 		"ci": [
 			"composer validate --no-interaction",
-			"vendor/bin/phpunit"
+			"vendor/bin/phpunit",
+			"@phpcs"
```

Comment:
ah, no, forget the question. I see this the reason of CI build's failure. So this line should be changed to `composer phpcs`, right?